### PR TITLE
Add telemetry to toggle line comment.

### PR DESCRIPTION
**Pull request by dibarbet**

### CODE REVIEW

File: src/EditorFeatures/Core/Implementation/CommentSelection/AbstractCommentSelectionBase.cs [hunk 1]
```diff
@@ -37,6 +37,9 @@ internal enum Operation
 
     internal abstract class AbstractCommentSelectionBase<TCommand>
     {
+        protected const string LanguageNameString = "languagename";
```
Comment (heejaechang) [inline src/EditorFeatures/Core/Implementation/CommentSelection/AbstractCommentSelectionBase.cs]: don't you have same thing defined in comment block? share the code?

Comment (dibarbet) [reply src/EditorFeatures/Core/Implementation/CommentSelection/AbstractCommentSelectionBase.cs]: That's why I moved it, so I could reuse it :)  This is the same one being used in toggle block

**Comment by dibarbet** [comment]
@jinujoseph or @vatsalyaagrawal for approval

**Comment by jinujoseph** [comment]
@dibarbet you may have to port this change over to d16.1 branch

**Pull request merged**